symengine/add.h

```diff
@@ -67,6 +67,10 @@ class Add : public Basic {
     virtual vec_basic get_args() const;
 
     virtual void accept(Visitor &v) const;
+
+    inline RCP<const Number> get_coef_() const { return coef_; }
+
+	inline umap_basic_num get_dict_() const { return dict_; }    
```

Comment:
Return a const reference to `umap_basic_num` instead of copying the dictionary here